packages/esm-appointments-app/src/header/appointments-header.component.tsx

```diff
@@ -37,20 +57,13 @@ const AppointmentsHeader: React.FC<AppointmentHeaderProps> = ({ title, appointme
           />
         </DatePicker>
         {typeof onChange === 'function' && (
-          <Dropdown
-            aria-label={t('selectServiceType', 'Select service type')}
-            className={styles.dropdown}
-            direction="bottom"
-            id="serviceDropdown"
-            items={[{ name: 'All', uuid: '' }, ...serviceTypes]}
-            itemToString={(item) => (item ? item.name : '')}
-            label={t('selectServiceType', 'Select service type')}
-            onChange={({ selectedItem }) => onChange(selectedItem?.uuid)}
-            selectedItem={
-              serviceTypes.find((service) => service.uuid === appointmentServiceType) || { name: 'All', uuid: '' }
-            }
-            size="sm"
-            titleText={t('view', 'View')}
+          <MultiSelect
+            id="serviceTypeMultiSelect"
+            label={t('filterByServiceType', 'Filter by service type')}
```

Comment:
```suggestion
            label={t('filterAppointmentsByServiceType', 'Filter appointments by service type')}
```

I propose changing the MultiSelect label for two reasons:

1. The suggested label is longer which means the dropdown menu is larger, giving us more room for the filter labels.
2. The suggested label is arguably clearer about communicating what this filter does to the user.